packages/twenty-front/src/modules/workflow/components/nodes/CreateStepNode.tsx

```diff
@@ -0,0 +1,22 @@
+import { IconButton } from '@/ui/input/button/components/IconButton';
```

Comment:
by project convention when we create subfolder we do it the other way: packages/twenty-front/src/modules/workflow/nodes/components/CreateStepNode.tsx

The idea is that all leaves folder respect the same structure of components/hooks/types, ...

Reply:
I chose to flatten the components directory instead. I don't think having a `nodes` submodule is essential.